Update native and cross libtool package revisions for driver flag handling.
Origin: EmberBSD (AI-assisted).

--- a/devel/libtool-base/Makefile
+++ b/devel/libtool-base/Makefile
@@ -31,7 +31,7 @@
 .include "../../devel/libtool/Makefile.common"
 
 PKGNAME=	${DISTNAME:S/-/-base-/}
-PKGREVISION=	3
+PKGREVISION=	4
 
 COMMENT=	Generic shared library support script (the script itself)
 
--- a/cross/cross-libtool-base/Makefile
+++ b/cross/cross-libtool-base/Makefile
@@ -38,7 +38,7 @@
 # XXX Tweaked for cross-compilation.
 #PKGNAME=	${DISTNAME:S/-/-base-/}
 PKGNAME=	${DISTNAME:S/^libtool-/cross-libtool-base-${MACHINE_PLATFORM}-/}
-PKGREVISION=	3
+PKGREVISION=	4
 
 COMMENT=	Generic shared library support script (the script itself)
 
--- a/devel/libtool-fortran/Makefile
+++ b/devel/libtool-fortran/Makefile
@@ -33,7 +33,7 @@
 FILESDIR=	${.CURDIR}/../libtool-base/files
 
 PKGNAME=	${DISTNAME:S/-/-fortran-/}
-PKGREVISION=	2
+PKGREVISION=	3
 
 COMMENT=	Generic shared library support script (the script itself, incl. Fortran)
 
--- a/devel/libtool/Makefile
+++ b/devel/libtool/Makefile
@@ -1,11 +1,12 @@
 # $NetBSD: Makefile,v 1.53 2022/07/26 16:15:26 wiz Exp $
 
 META_PACKAGE=	yes
+PKGREVISION=	1
 
 .include "Makefile.common"
 
-DEPENDS+=	${DISTNAME:S/-/-base-/}{,nb[0-9]*}:../../devel/libtool-base
-DEPENDS+=	${DISTNAME:S/-/-fortran-/}{,nb[0-9]*}:../../devel/libtool-fortran
+DEPENDS+=	libtool-base>=${PKGVERSION_NOREV}nb4:../../devel/libtool-base
+DEPENDS+=	libtool-fortran>=${PKGVERSION_NOREV}nb3:../../devel/libtool-fortran
 DEPENDS+=	${DISTNAME:S/-/-info-/}{,nb[0-9]*}:../../devel/libtool-info
 
 # There is no libltdl dependency here; this is deliberate, so that libltdl
